setup-gcloud/README.md

```diff
@@ -63,6 +63,7 @@ steps:
     project_id: ${{ secrets.GCP_PROJECT_ID }}
     service_account_key: ${{ secrets.GCP_SA_KEY }}
     export_default_credentials: true
+    credentials_file_path: '/tmp/gcloud-credentials'
```

Comment:
nit: I think it's better if we do not show this for our example invocation. The standard behavior should be not setting this.

Reply:
Perhaps we can showcase this if needed as an alternate config like here?
https://github.com/GoogleCloudPlatform/github-actions/tree/master/get-secretmanager-secrets#authorization

Reply:
done.